src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
- 1: LoadtestTasks        |
-   1:                    | 2,054          | 0 (0%)         | 410.8  | 0.000
--------------------------------------------------------------------------------
- Name                    | Avg (ms)   | Min        | Max        | Median
- -----------------------------------------------------------------------------
- 1: LoadtestTasks        |
-   1:                    | 20.86      | 7          | 254        | 19
-
-=== PER REQUEST METRICS ===
-------------------------------------------------------------------------------
- Name                    | # reqs         | # fails        | req/s  | fail/s
- -----------------------------------------------------------------------------
- GET /                   | 2,054          | 0 (0%)         | 410.8  | 0.000
--------------------------------------------------------------------------------
- Name                    | Avg (ms)   | Min        | Max        | Median
- -----------------------------------------------------------------------------
- GET /                   | 20.68      | 7          | 254        | 19
--------------------------------------------------------------------------------
- Slowest page load within specified percentile of requests (in ms):
- ------------------------------------------------------------------------------
- Name                    | 50%    | 75%    | 98%    | 99%    | 99.9%  | 99.99%
- -----------------------------------------------------------------------------
- GET /                   | 19     | 21     | 53     | 69     | 250    | 250
+$ cargo run --release -- --host https://drupal-memcache.ddev.site --report-file=report.html
+    Finished release [optimized] target(s) in 0.17s
+     Running `target/release/goose_attack --host 'https://drupal-memcache.ddev.site' --report-file=report.html`
+
+// Some output omitted...
 ```
 
-By default, Goose will hatch 1 `GooseUser` per second, up to the number of CPU cores available on the server used for load testing. In the above example, the server has 8 CPU cores, so it took 8 seconds to hatch all users. After all users are hatched, Goose flushes all metrics collected during the hatching process so all subsequent metrics are taken with all users running. Before flushing the metrics, they are displayed to the console so the data is not lost.
+The HTML report will include a few sections. Let's go through them: 
+
+![HTML report header section](report_header.png)
+
+By default, Goose will hatch 1 `GooseUser` per second, up to the number of CPU cores available on the server used for load testing. In the above example, the server has 8 CPU cores, so it took 8 seconds to hatch all users. After all users are hatched, Goose flushes all metrics collected during the hatching process (use `--no-reset-metrics` to disable this behavior) so all subsequent metrics are taken with all users running. Before flushing the metrics, they are displayed to the console so the data is not lost.
```

Comment:
In the above screenshot it's actually showing a startup time of 7 seconds, not 8, which is confusing as it doesn't match what we say. Is that bad luck, and can you just run it again to capture a screenshot that matches the expectation?

Reply:
I tried to run it a few more times, but I am getting the same result every time. Thinking about it more it actually makes sense. If there are 8 users to be hatched and the first one is hatched at the beginning of the test the last one will be hatched around the end of the 7th second. Maybe this logic changed slightly since this was last updated?

I updated the text to reflect that.

Reply:
Yes, the math works: thanks for clarifying in the docs.